### PR TITLE
os/bluestore: fix latency calculation

### PR DESCRIPTION
**Pull request by liupan1111**
add two statistic variables in bluestore to cal commit latency and apply latency.

Signed-off-by: Pan Liu <pan.liu@istuary.com>

### CODE REVIEW

**Comment by liupan1111** [comment]
@liewegas, @tchaikov , please help take a look. Thanks.

**Comment by tchaikov** [comment]
@liupan1111 could you remove the merge commit from your PR?

**Comment by liupan1111** [comment]
@tchaikov, please give me some guidence. I could  use git push -f to avoid the merge commit as the last commit.But if it happens as the first commit because I merge newest from ceph/master to my forked, I cannot find a way to remove it...

**Comment by tchaikov** [comment]
@liupan1111 how about cherry-picking e0afcfd and 37822cd in a brand new branch sync'ed with ceph/master, then "git push -f"?

File: src/osd/osd_types.cc [hunk 1]
```diff
@@ -322,8 +322,8 @@ void osd_stat_t::dump(Formatter *f) const
   f->open_object_section("op_queue_age_hist");
   op_queue_age_hist.dump(f);
   f->close_section();
-  f->open_object_section("fs_perf_stat");
-  fs_perf_stat.dump(f);
+  f->open_object_section("perf_stat");
+  os_perf_stat.dump(f);
```
Comment (liewegas) [inline src/osd/osd_types.cc]: Please put a note in PendingReleaseNotes that the json schema has changed.

File: src/os/bluestore/BlueStore.cc [hunk 2]
```diff
@@ -2733,6 +2733,10 @@ void BlueStore::_init_logger()
     "Average finishing state latency");
   b.add_time_avg(l_bluestore_state_done_lat, "state_done_lat",
     "Average done state latency");
+  b.add_time_avg(l_bluestore_ack_lat, "ack_lat",
+    "Average ack latency");
+  b.add_time_avg(l_bluestore_commit_lat, "commit_lat",
+    "Average commit latency");
```
Comment (liewegas) [inline src/os/bluestore/BlueStore.cc]: No need for 2 of these; ack and commit callbacks are triggered at the same time for bluestore

Comment (liewegas) [inline src/os/bluestore/BlueStore.cc]: Let's just do commit_lat

File: src/osd/osd_types.cc (hunk 1 above)
Comment (liupan1111) [inline src/osd/osd_types.cc]: Sure.

**Comment by liupan1111** [comment]
@liewegas @tchaikov , I've modified as your suggestion, please help review. Thanks.

**Pull request merged**